build_product_images.py

```diff
@@ -85,26 +92,55 @@ def build_and_publish_image(args, product):
     """
     Returns a list of commands that need to be run in order to build and
     publish product images.
+
+    For local building, builder instances are supported.
     """
     commands = []
-
+    push = ""
     image_name = f'{args.registry}/stackable/{product["name"]}'
     tags = build_image_tags(image_name, args.image_version, args.product_version)
     build_args = build_image_args(product["versions"][0])
 
-    commands.append(
-        [
-            "docker",
-            "build",
-            *build_args,
-            *tags,
-            "-f",
-            product["name"] + "/Dockerfile",
-            ".",
-        ]
-    )
     if args.push:
-        commands.append(["docker", "push", "--all-tags", image_name])
+        push = "--push"
+
+    # Multiarch builds
+    if len(args.architecture) > 1:
```

Comment:
Would be fine by me, downside would be that if a builder is up and running because somebody did `docker buildx create --use`, without `--load` the image will not be loaded into the registry. On a fresh environment, that would never be the case I guess and docker will give a hint: `WARNING: No output specified for docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load` .

This might be hint enough, if we do not care about this I am happy to remove it :)